cmd/registry/controller/controller.go

```diff
@@ -286,7 +286,7 @@ func generateCreateActions(
 		}
 		parentList = []patterns.ResourceInstance{
 			patterns.ProjectResource{
-				ProjectName: parentName,
+				Project: &rpc.Project{Name: parentName.String()},
```

Comment:
Note that "Project" is only defined in the Admin service.

Reply:
I understand the confusion, `ProjectResource` is only used in this particular instance and we are mainly interested in it's name. At no time is the `rpc.Project` resource being fetched from the registry, since the controller is the registry client command.

To mimimize the confusion, I removed the `rpc.Project` resource from the struct, we still need to keep the timestamp function (which returns null value), for `ProjectResource` to be able to inherit `ResourceInstance`.

See the latest commit `Handle ProjectResource properly` for the fix I made.